src/components/picker/api/picker.api.json

```diff
@@ -108,13 +108,403 @@
   ],
   "snippet": [
     "<Picker",
-    "  value={currentValue$1}",
+    "  value={2$1}",
     "  placeholder={'Placeholder'$2}",
     "  onChange={() => console.log('changed')$3}",
+    " items={[{label: 'Item 1', value: 1}, {label: 'Item 2', value: 2},{label: 'Item 3', value: 3}]}",
     ">",
-    "  {_.map(items, item => (",
-    "   return renderItem(item, index);",
-    "  ))$4}",
     "</Picker>"
-  ]
+  ],
+  "docs": {
+    "hero": {
+      "title": "Picker",
+      "description": "Picker allows the user to select an option from a variety of choices available.Pressing the Picker Field, will open a dialog or modal displaying a list of available options for selection.",
+      "type": "hero",
+      "layout": "horizontal",
+      "content": [
+        {
+          "value": "https://wixmp-1d257fba8470f1b562a0f5f2.wixmp.com/mads-docs-assets/assets/Components%20Docs/Picker/picker_cover.png"
+        }
+      ]
+    },
+    "tabs": [
+      {
+        "title": "Overview",
+        "sections": [
+          {
+            "type": "section",
+            "title": "Usage Example",
+            "description": "Pickers allow selection of an option or multiple options from a list usually displayed using an actionSheet. Pickers can be used in forms, settings screens or as part of an interactive flow.",
+            "content": [
+              {
+                "value": "https://embed.figma.com/design/xFjvYNkGTmYTGYMLrmz9Ir/Guidelines-to-Docs?node-id=4415-185562&embed-host=share"
+              }
+            ]
+          },
+          {
+            "type": "list",
```

Comment:
I think we're missing a preset example since we have the default, underline and outline presets here and may consider removing preset=outline from snippets to keep default. WDYT?